src/bilder/images/edxapp/packer_vars/mitx.pkrvars.hcl

```diff
@@ -0,0 +1,3 @@
+build_environment    = "mitx-ci"
```

Comment:
Does the filename need to include the `ci` in it?

Reply:
No, because the AMI only gets built once and used across environments. We're just using the CI build environment so that it doesn't run any risk of interfering with other environments.